src/fabric/test/fabric2_doc_crud_tests.erl

```diff
@@ -19,55 +19,58 @@
 -include("fabric2.hrl").
 
 
+-define(TDEF(Name), {atom_to_list(Name), fun Name/1}).
```

Comment:
Could we move this into the couch_eunit.hrl file so we can use it everywhere. And also move the `with` function somewhere so we don't have to keep repeating it.

Reply:
I contemplated something along those lines but couldn't convince myself on the best place to put it. I avoided couch_eunit.hrl because that could spider out to a number of other tests causing unneeded code churn. I'll see if a .hrl in the fabric test directory works.

Reply:
Done.